### ISSUE TITLE
Images are pixelated in kitty terminal

### ISSUE THREAD

**Issue by stmcginnis** [mfontanini/presenterm#267]
This is likely something in my local configuration, but hoping someone here might know what that could be.

I can display images in kitty fine, and they look like I would expect:

![image](https://github.com/mfontanini/presenterm/assets/3922475/9171c7cb-93be-4cad-8e5c-75c7799e68f2)

But when I try to display images in a presentation, they are rendered horribly pixelated:

![image](https://github.com/mfontanini/presenterm/assets/3922475/3c91e1d5-9d14-4d03-971b-18375dd3961b)

Am I missing some configuration or setting needed to make this rendering work well?

---

Kitty version: 0.35.1
Presenterm version: 0.7.0

**Comment by stmcginnis** [comment]
OK, turned out I needed to explicitly add `--image-protocol kitty-local` when launching. Is this expected? I assumed it would automatically detect that, but maybe something else is still missing?

**Comment by mfontanini** [comment]
Are you using tmux? It seems like the kitty detection is failing, which is a known issue under tmux. Only in that scenario you currently need to explicitly use `--image-protocol`, so you shouldn't need to set it unless you're using it.

You can make it the default in the [config file](https://mfontanini.github.io/presenterm/guides/configuration.html#preferred-image-protocol) so you don't have to pass it in every time if you are actually using tmux.

**Comment by stmcginnis** [comment]
I was not using tmux, and I opened a fresh instance of kitty just to make sure there wasn't some sort of leftover settings in my environment.

I did get around it by explicitly setting `kitty-local` in the config file, so things seem to be working well for me now. But if there are anything that would be useful to debug from this environment, just let me know. I'll close this issue for now.

Thanks for such a great tool!
